corehq/apps/app_manager/static/app_manager/js/download_async_modal.js

```diff
@@ -13,65 +13,81 @@ $(function(){
         self.$download_progress = self.$el.find("#" + self.el_id + "-download-progress");
         self.$downloading = self.$el.find("#" + self.el_id + "-downloading");
 
+        self.download_in_progress = false;
```

Comment:
you mean here and in `init` - no reason I guess, can remove them from here and leave them in `init`